tabbycat/draw/generator/powerpair.py

```diff
@@ -79,6 +81,9 @@ def __init__(self, *args, **kwargs):
         if self.options["pullup_restriction"] == "least_to_date":
             self.check_teams_for_attribute("npullups", checkfunc=lambda x: isinstance(x, int))
 
+        if self.options["pullup_restriction"] == "easy_draw":
+            self.check_teams_for_attribute("drawstrength", checkfunc=lambda x: isinstance(x, float))
```

Comment:
When generating the draw, I get the error: "24 out of 24 teams don't have a 'drawstrength' attribute." Is this meant to be "`draw_strength_score`" instead of `"drawstrength"`?